core/src/main/java/org/elasticsearch/rest/RestController.java

```diff
@@ -308,6 +309,23 @@ private boolean hasContentTypeOrCanAutoDetect(final RestRequest restRequest, fin
         return true;
     }
 
+    private String parseMediaType(String rawContentType) {
+        final String contentType = rawContentType.toLowerCase(Locale.ROOT);
+        int firstSemiColonIndex = contentType.indexOf(';');
```

Comment:
make it `final`

Reply:
Thanks for picking this up. I pushed 0fabbd002c47ef86fb0c0eda01f184e6fe456171 . The merge target is 5.6 now. Thank you so much!